jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml

```diff
@@ -179,6 +179,60 @@
           timeout --signal=ALRM 150m bash -x -c ". scripts/qa_crowbarsetup.sh ; onadmin_runlist $commands;"
           ' || ret=$?
 
+          #Test nova evacuate
+          OUTPUT=$(ssh -T controller1 <<'EOF' 
+          source /root/.openrc
+          # Get hypervisor name
+          hypervisor_host=$(nova hypervisor-list | grep enabled | grep 1 |cut -d '|' -f 3 | cut -d '.' -f 1 |cut -d ' ' -f 2)
+          echo $hypervisor_host
+          export hypervisor_host=$hypervisor_host
+          # Crate instance on specific hypervisor image always exist after tempest run (cirros-0.3.4-x86_64-tempest-machine)
+          nova boot --image cirros-0.3.4-x86_64-tempest-machine  --flavor  tempest-stuff  --availability-zone nova:$hypervisor_host NovaEvacuate
+
+          #Check instance is ACTIVE and Running
+          status=$(nova show  NovaEvacuate | grep status | cut -d '|' -f 3 | grep ACTIVE)
+          echo $status
+          if [ $status = "ACTIVE" ] ; then
+           echo "Instance launched successfully"
+          else
+           echo  "Instance luanch failed"
+          fi
+          EOF
+          )
+          hypervisor_host=$(echo $OUTPUT | cut -d ' ' -f 1)
+
+          #Get full hypervisor hostname
+          echo "$hypervisor_host"  > temp
+          hypervisor_fqdn=$(crowbarctl node list | grep $hypervisor_host | cut -d '|' -f 2)
+          scp temp controller1:/root/
+          #power off hypervisor node
+          crowbarctl node reboot $hypervisor_fqdn
+          #wait
+          sleep 180
+
+          #Check that instance evacuated
+
+          ssh -T controller1  << 'EOF'
+
+          hypervisor_host=$(cat /root/temp)
+          source /root/.openrc
+          #Check that instance evacuated to working hypervisor host
+          hypervisor_evacuate_host=$(nova show  NovaEvacuate | grep hypervisor_hostname | cut -d '|' -f 3 | cut -d '.' -f 1 |cut -d ' ' -f 2)
```

Comment:
extra space before NovaEvacuate, there are also too many cut -f, can you write a text example of the output in the comment?